lib/ast.js

```diff
@@ -989,7 +989,8 @@ var AST_ObjectProperty = DEFNODE("ObjectProperty", "key value", {
     $documentation: "Base class for literal object properties",
     $propdoc: {
         key: "[string|AST_Node] the property name converted to a string for ObjectKeyVal. For setters, getters and computed property this is an arbitrary AST_Node",
-        value: "[AST_Node] property value. For setters and getters this is an AST_Function."
+        value: "[AST_Node] property value. For setters and getters this is an AST_Function.",
+        default: "[AST_Expression] The default for this parameter, only used when nested inside a binding pattern"
```

Comment:
They take over these properties no? (if my reference is the same as yours though)